sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -128,18 +128,10 @@ def __init__(self, init_config, instance_config, service_check_handler):
         self.existing_databases = None
         self.server_version = int(self.instance.get('server_version', self.DEFAULT_SQLSERVER_VERSION))
 
-        self.adoprovider = self.default_adoprovider
-
-        self.valid_connectors = []
-        if adodbapi is not None:
-            self.valid_connectors.append('adodbapi')
-        if pyodbc is not None:
-            self.valid_connectors.append('odbc')
-
+        self.valid_connectors = ['adodbapi', 'odbc']
```

Comment:
Do we need to check if adodbapi is installed like it did before? The dependency states that it's only relevant for windows https://github.com/DataDog/integrations-core/blob/bdccb8fb664514060aac782748b00199a61c4a60/sqlserver/pyproject.toml#L40